src/lib/Runner.ts

```diff
@@ -68,6 +68,7 @@ export class Runner {
   };
 
   moveFromQueueToRunning = async (landRequestStatus: LandRequestStatus, lockId: Date) => {
+    if (await this.getPauseState()) return;
```

Comment:
should we hold this value in a variable instead?

Reply:
We still need to put the variable in redis, which might be a bit quicker than db. But the `PauseState` is an one-row table, I guess the query is not that slow and we won't get a drastic speed increase by moving to redis.